salt/states/keystone.py

```diff
@@ -173,11 +182,11 @@ def user_present(name,
             ret['comment'] = 'User "{0}" has been updated'.format(name)
             ret['changes']['Password'] = 'Updated'
         if roles:
-            for tenant_role in roles:
+            for tenant in roles.keys():
```

Comment:
We should not need to call .keys(), just iterating over the dict should return the keys